docs/docs/rad-lab-ui/ui_installation/source-control.md

```diff
@@ -5,10 +5,16 @@ title: 01 - Source Control
 
 # Source Control
 
-We recommend forking the official [RAD Lab repository](https://github.com/GoogleCloudPlatform/rad-lab), to ensure that your organization keeps full control over the codebase.  For now, RAD Lab only supports GitHub, but this may change in the future.  When pushing the code to GitHub, ensure that it's configured to work together with the [Cloud Build integration](https://github.com/marketplace/google-cloud-build).  
+Users either have the option to fork the official [RAD Lab repository](https://github.com/GoogleCloudPlatform/rad-lab) or to clone the existing repository and push the code to a repository they manage and control.  The problem is that forking a repo results in a public repository, which may violate existing security policies in your organization.  
```

Comment:
Can we state that we recommend to **clone the repo** and **move it to their own Private repo**?